src/env_properties.h

```diff
@@ -26,6 +26,11 @@
   V(js_transferable_wrapper_private_symbol, "node:js_transferable_wrapper")    \
   V(entry_point_module_private_symbol, "node:entry_point_module")              \
   V(entry_point_promise_private_symbol, "node:entry_point_promise")            \
+  V(module_source_private_symbol, "node:module_source")                        \
+  V(module_export_names_private_symbol, "node:module_export_names")            \
+  V(module_circular_visited_private_symbol, "node:module_circular_visited")    \
+  V(module_export_private_symbol, "node:module_export")                        \
+  V(module_parent_private_symbol, "node:module_parent")                        \
```

Comment:
There are module-related symbols exported from `node_contextify.cc` too, should these come from there?

Reply:
At the moment, the convention is to declare private symbols in this per-isolate list because they are primitive values that can be shared across contexts. Moving them to files like `node_contextify.cc` works but it adds more boilerplate codes (https://github.com/nodejs/node/blob/main/src/node_util.cc#L281-L295).

Reply:
I don't think there are any private symbols created in `node_contextify.cc`, but if they are, they should be moved here instead.